calm/samples/traderx/traderx.json

```diff
@@ -118,153 +118,169 @@
         "nodes": [
           "web-client"
         ]
-      }
+      },
+      "relationship-t": ""
     },
     {
       "uniqueId": "web-client-uses-web-gui",
-      "relationship-type": "connects",
-      "parties": {
-        "source": "web-client",
-        "destination": "web-gui-process"
+      "relationship-type": {
+        "connects": {
+          "source": "web-client",
+          "destination": "web-gui-process"
+        }
       },
       "protocol": "HTTPS",
       "authentication": "OAuth2"
     },
     {
       "uniqueId": "web-gui-uses-position-service",
-      "relationship-type": "connects",
-      "parties": {
-        "source": "web-gui-process",
+      "relationship-type": {
+        "connects": {
+
+        },  "source": "web-gui-process",
         "destination": "position-service"
       },
       "protocol": "HTTPS",
       "authentication": "SPNEGO"
     },
     {
       "uniqueId": "position-service-uses-traderx-db",
-      "relationship-type": "connects",
-      "parties": {
-        "source": "position-service",
-        "destination": "traderx-db"
+      "relationship-type": {
+        "connects": {
+          "source": "position-service",
+          "destination": "traderx-db"
+        }
       },
       "protocol": "JDBC",
       "authentication": "Kerberos"
     },
     {
       "uniqueId": "traderx-system-is-deployed-in-internal-bank-network",
-      "relationship-type": "deployed-in",
-      "parties": {
-        "nodes": [
-          "traderx-system"
-        ],
-        "container": "internal-bank-network"
+      "relationship-type": {
+        "deployed-in": {
+          "nodes": [
+            "traderx-system"
+          ],
+          "container": "internal-bank-network"
+        }
       }
     },
     {
       "uniqueId": "traderx-system-is-composed-of",
-      "relationship-type": "composed-of",
-      "parties": {
-        "nodes": [
-          "web-client",
-          "web-gui-process",
-          "position-service",
-          "traderx-db"
-        ],
-        "container": "traderx-system"
-      }
+      "relationship-type": {
+        "composed-of": {
+          "nodes": [
+            "web-client",
+            "web-gui-process",
+            "position-service",
+            "traderx-db"
+          ],
+          "container": "traderx-system"
+        }
+      },
     },
     {
       "uniqueId": "traderx-system-components-are-deployed-in-internal-bank-network",
-      "relationship-type": "deployed-in",
-      "parties": {
-        "nodes": [
-          "web-client",
-          "web-gui-process",
-          "position-service",
-          "traderx-db"
-        ],
-        "container": "internal-bank-network"
-      }
+      "relationship-type": {
+        "deployed-in": {
```

Comment:
Minor point but should we change the deployed-in to show the container first, does it read more naturally?